features/internal_destinations.feature

```diff
@@ -90,22 +90,6 @@ Feature: Internal Destinations
         """
         Then we get OK response
 
-        When we get "/archive"
-        Then we get list with 1 items
```

Comment:
can you rather change it to `we get list with 0 items` so it's obvious this was intended